trytond/model/modelsql.py

```diff
@@ -1457,34 +1459,31 @@ def _update_mptt(cls, field_names, list_ids, values=None):
         cursor = Transaction().connection.cursor()
         for field_name, ids in zip(field_names, list_ids):
             field = cls._fields[field_name]
-            if (isinstance(field, fields.Many2One)
-                    and field.model_name == cls.__name__
-                    and field.left and field.right):
-                if (values is not None
-                        and (field.left in values or field.right in values)):
-                    raise Exception('ValidateError',
-                        'You can not update fields: "%s", "%s"' %
-                        (field.left, field.right))
-
-                # Nested creation require a rebuild
-                # because initial values are 0
-                # and thus _update_tree can not find the children
-                table = cls.__table__()
-                parent = cls.__table__()
-                cursor.execute(*table.join(parent,
-                        condition=Column(table, field_name) == parent.id
-                        ).select(table.id,
-                        where=(Column(parent, field.left) == 0)
-                        & (Column(parent, field.right) == 0),
-                        limit=1))
-                nested_create = cursor.fetchone()
-
-                if not nested_create and len(ids) < 2:
-                    for id_ in ids:
-                        cls._update_tree(id_, field_name,
-                            field.left, field.right)
-                else:
-                    cls._rebuild_tree(field_name, None, 0)
+            if (values is not None
```

Comment:
Ilm anque un check sur field in mptt, non ?

Reply:
_update_mptt n'est appelé qu'avec des champs MPTT en théorie (il me semble que c'est toujours le cas ici).